integrations/vite/resolvers.test.ts

```diff
@@ -1,143 +1,148 @@
 import { describe, expect } from 'vitest'
```

Comment:
This test still uses a global expect — this is probably fine but maybe we should use the one from the test for consistency.